exchange/utils.go

```diff
@@ -643,20 +643,15 @@ func mergeImpFPD(imp *openrtb2.Imp, fpd json.RawMessage, index int) error {
 	return nil
 }
 
-var allowedImpExtFields = map[string]interface{}{
-	openrtb_ext.AuctionEnvironmentKey:       struct{}{},
-	openrtb_ext.FirstPartyDataExtKey:        struct{}{},
-	openrtb_ext.FirstPartyDataContextExtKey: struct{}{},
-	openrtb_ext.GPIDKey:                     struct{}{},
-	openrtb_ext.SKAdNExtKey:                 struct{}{},
-	openrtb_ext.TIDKey:                      struct{}{},
-}
-
 var allowedImpExtPrebidFields = map[string]interface{}{
 	openrtb_ext.IsRewardedInventoryKey: struct{}{},
 	openrtb_ext.OptionsKey:             struct{}{},
 }
 
+var deniedImpExtFields = map[string]interface{}{
+	"prebid": struct{}{},
```

Comment:
Please extract to a variable here as we had in `allowedImpExtFields`, or use an existing variable if it exists already.